File: other/tokilearning/Latihan/procedure.cpp
#include <stdio.h>

using namespace std;

void tulis(int n);

int main(){
    int n;
    
    while (scanf("%d\n",&n)!=EOF) tulis(n);
    
    return 0;
}

void tulis(int n){
    if (n<10) printf("satuan\n");
      else if (n<100) printf("puluhan\n");
        else if (n<1000) printf("ratusan\n");
          else if (n<10000) printf("ribuan\n");
            else printf("puluhribuan\n");
}

File: other/tokilearning/Latihan/procedure.pas
var
  x:integer;

procedure TulisJawaban(x: integer);
begin
    case x of
        1..9: begin writeln('satuan'); end;
        10..99: begin writeln('puluhan'); end;
        100..999: begin writeln('ratusan'); end;
        1000..9999: begin writeln('ribuan'); end;
        10000..30000: begin writeln('puluhribuan'); end;
    end;
end;

begin
  while not eof do
    begin
      readln(x);
      TulisJawaban(x);
    end;
end.